math/src/field/f62/mod.rs

```diff
@@ -286,6 +286,37 @@ impl Neg for BaseElement {
     }
 }
 
+// QUADRATIC EXTENSION
+// ================================================================================================
+
+/// Defines a quadratic extension of the base field over an irreducible polynomial x<sup>2</sup> -
+/// x - 1. Thus, an extension element is defined as α + β * φ, where φ is a root of this polynomial,
+/// and α and β are base field elements.
+impl ExtensibleField<2> for BaseElement {
```

Comment:
For `f64` quadratic extension field is defined over x<sup>2</sup> - x + 2 irreducible polynomial (rather than  x<sup>2</sup> - x - 1 here). It is exactly the same as for `f128` though because the same polynomial works for both `f62` and `f128` fields.